### ISSUE TITLE
Remove File/Directories if @import statement removed

### ISSUE THREAD

**Issue by una** [Stephn-R/sass-director-sublime#4]
Just an idea for an enhancement :) You're doing an awesome job.

**Comment by Stephn-R** [comment]
Thank you, this is an interesting idea. Perhaps we can consider doing the following:

1. User chooses root folder of their Sass Project
2. Selects all sass related files
3. Maps any dependencies
4. If a file has no imports (and is not a manifest file) --> ***Ask for Removal***

We can also call this an ```Interactive Manifest Update```

**Comment by una** [comment]
Well what I mean't was, if the file is taken out of the manifest, i.e:

Before:
```
@import 'dir1/dir2/fileA`
@import 'dir1/dir2/fileB`
```

After:
```
@import 'dir1/dir2/fileA`
```

Should reflect the update and remove fileB

**Comment by Stephn-R** [comment]
Yes, the method I explained would be an interactive update. But we could definitely speed up some of the time by doing an automated update
